recipes/qt-msbuild/all/conanfile.py

```diff
@@ -0,0 +1,39 @@
+import os
+
+from conan import ConanFile
+from conan.tools.files import copy, get
+from conan.tools.layout import basic_layout
+
+required_conan_version = ">=1.53.0"
+
+class QtMSBuildConan(ConanFile):
+    name = "qt-msbuild"
+    description = "Qt/MSBuild MSBuild rules and targets from Qt VS Tools"
+    license = "LicenseRef-Qt-Commercial OR GPL-3.0-only WITH Qt-GPL-exception-1.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.qt.io"
+    topics = ("qt", "msbuild")
+
+    package_type = "build-scripts"
+    settings = "os", "arch", "compiler", "build_type"
```

Comment:
I've noticed that other similar recipes always specify settings "os", "arch", "compiler", "build_type" even though I don't understand why.

Since the packaged contents are build scripts that are independent of any os/arch/build type, I think those settings could be removed?